### PR TITLE
fix: add uglifyOptions

### PR DESCRIPTION
**Pull request by montezume**
This fixes the production build of most apps.

### CODE REVIEW

**Comment by montezume** [comment]
I'm not sure if this is the issue. I'm having problems with one of the graphql requests returning HTML instead of JSON 🤔

File: packages/mc-scripts/config/create-webpack-config-for-production.js [hunk 1]
```diff
@@ -22,6 +22,22 @@ const postcssColorModFunction = require('postcss-color-mod-function');
...
+    warnings: false,
+    parse: {},
+    compress: {},
+    mangle: true,
+    output: null,
+    toplevel: false,
+    nameCache: null,
+    ie8: false,
+    // https://github.com/mishoo/UglifyJS2/issues/1609
+    collapse_vars: false,
```
Comment (montezume) [inline packages/mc-scripts/config/create-webpack-config-for-production.js]: this fixing a problem in a few apps

File: packages/mc-html-template/index.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = templateParams => {
 
   const cssChunks = cssVendorChunks.concat(cssAppChunks).map(fileName => {
     const chunkPath = fileName.replace(/^\//, '');
-    return `<link href="__CDN_URL__${chunkPath}" rel='stylesheet' type='text/css'></link>`;
+    return `<link href="__CDN_URL__${chunkPath}" rel='stylesheet' type='text/css'>`;
```
Comment (montezume) [inline packages/mc-html-template/index.js]: this doesn't give a warning in IE

File: packages/mc-scripts/config/create-webpack-config-for-production.js (hunk 1 above)
Comment (emmenko) [reply packages/mc-scripts/config/create-webpack-config-for-production.js]: So we're adding back what we had before? 🤔

Comment (montezume) [reply packages/mc-scripts/config/create-webpack-config-for-production.js]: sort of, what we had before was for a different uglifer

File: packages/mc-html-template/index.js (hunk 2 above)
Comment (montezume) [reply packages/mc-html-template/index.js]: https://stackoverflow.com/questions/9943981/can-self-closing-link-tags-be-problematic

**Comment by montezume** [comment]
I'm going to close this for now in favour of patching the webpack config in the apps

**Pull request closed without merging**